src/Vcard.php

```diff
@@ -105,6 +109,34 @@ public function gender(string $gender): self
         return $this;
     }
 
+    public function org(string $org): self
```

Comment:
Good point. It wasn't clear what the example `ORG:Google;GMail Team;Spam Detection Squad` means, as there is no explicit description that it is `company`, `unit`, `team`. It seems correct. I'll change that to.

If the `unit` is missing is the output then
* `ORG:Google;;Spam Detection Squad` (if a prettymuch for this ) OR
* `ORG:Google;Spam Detection Squad` (here it would be unclear what the unit and what the team is) ?

Reply:
Or we just pass empty strings 😅 ... ignore my question ^^

Reply:
The arguments are not optional but could be empty strings so any combination is possible. 

* `ORG:;;team`
* `ORG:;unit;team`
* `ORG:company;unit;team`
* `ORG:company;;team`
* `ORG:company;unit`

Reply:
Hey, 
Optional in sense that you don't have to define unit & team. Only company is required.
I would prefer `null` as default - so `?string` as typehint.
And the output would be `ORG: Google;;Team`.